sdk/synapse/Microsoft.Azure.Management.Synapse/src/Generated/Models/Workspace.cs

```diff
@@ -106,6 +108,7 @@ public Workspace(string location, string id = default(string), string name = def
             CspWorkspaceAdminProperties = cspWorkspaceAdminProperties;
             Settings = settings;
             AzureADOnlyAuthentication = azureADOnlyAuthentication;
+            TrustedServiceBypassEnabled = trustedServiceBypassEnabled;
```

Comment:
this actually causes breaking change, consider to always **append** optional parameters at the end, instead of inserting.